kythe-browse.sh

```diff
@@ -15,17 +15,20 @@
 
 usage() {
   cat <<EOF
-$0 verilog file...
+$0 PATH DIR INCLUDE_DIRs
```

Comment:
Near the top of the script, validate number of positional arguments to this script:

```shell
[[ "$#" == 3 ]] || { usage; exit 1; }
```